jobbergate-docs/src/local-development.rst

```diff
@@ -0,0 +1,263 @@
+   │ 78  │ converge-test          │ converge-app           │                                    │ abdallah.el-hajjam@scania.com      │ True                 │
+   │ 84  │ 00_smallcase           │ None                   │ Small application for testing      │ james.beedy@scania.com             │ True                 │
+   │     │                        │                        │ jobbergate.                        │                                    │                      │
+   ...
+   │ 91  │ tucker-test            │ tucker-test            │                                    │ tucker@omnivector.solutions        │ True                 │
+   └─────┴────────────────────────┴────────────────────────┴────────────────────────────────────┴────────────────────────────────────┴──────────────────────┘
+
+If you see this kind of output, congratulations! You now have an local instance of
+``jobbergate-cli`` working and ready for develoment.
+
+Make sure to deactivate the Poetry virtualenv before you move on by typing "exit" or
+hitting ``<ctl-d>``.
+
+
+Local jobbergate-api setup
+--------------------------
+
+The ``jobbergate-api`` is easiest to set up for local development using
+``docker-compose``. There is a ``docker-compose.yml`` in the root directory of the main
+``jobbergate`` repository that can be used to spin up the following services:
+
+* The ``jobbergate-api``
+* A local ``postgres`` instance for the ``jobbergate-api`` to consume
+* A test ``postgres`` instance to use for the unit tests
+* A ``minio`` instance for mimicing file storage on ``S3``
+
+
+Create some basic config
+........................
+
+First, ``docker-compose`` will need a ``.env`` file that contains some basic settings.
+Create the ``.env`` file in the root ``jobbergate`` directory with the following
+contents::
+
+   AUTH0_DOMAIN=omnivector.us.auth0.com
+   AUTH0_AUDIENCE=https://armada.omnivector.solutions
+
+
+Stand it up with ``docker-compose``
+...................................
+
+If you haven't already, `Clone the git repo`_.  From the root directory of the
+``jobbergate`` directory, execute ``docker-compose``:
+
+.. code-block:: bash
+
+   $ docker-compose up --build
+
+Wait until the compose finishes.
+
+
+Try out the API with Swagger
+............................
+
+Next we want to try out the API in a browser using Swagger. To do this, you will need
+a functioning auth token with Jobbergate permissions. The easiest way to do this is to
+use the ``jobbergate-cli``.
+
+Change to the ``jobbergate-cli`` directory and activate the Poetry shell again. Then
+show the cached auth token (you will need to login if you haven't yet):
+
+.. code-block:: bash
+
+   $ cd jobbergate-cli
+   $ poetry shell
+   $ jobbergate show-token --prefix --plain
+
+The token will automatically be copied to your clipboard for you. You can also just
+copy and paste it from the output if you like. You may now exit the Poetry shell and go
```

Comment:
Sorry, I didn't get where it should be pasted

Reply:
Added some explanation in commit af2b1fa